packages/vue-components/src/TipBox.vue

```diff
@@ -1,92 +1,109 @@
 <template>
-  <div :class="['alert box-container', containerStyle(), addClass]" :style="customStyle()">
-    <!-- Header wrapper, not rendered if there is no header attribute -->
-    <div v-if="headerBool()" :class="['box-header-wrapper', { 'alert-dismissible': dismissible }]">
-      <!-- icon on the left of the header -->
-      <div
-        v-if="iconBool()"
-        :class="['icon-wrapper', iconStyle()]"
-        :style="customIconColorStyle()"
-      >
-        <slot name="icon">
-          <i :class="['fas', getFontAwesomeIconStyle()]"></i>
-        </slot>
+  <div
+    :class="['alert box-container', containerStyle(), addClass]"
+    :style="customStyle()"
+  >
+    <div :class="[{ 'flex-row-layout': horizontalDividerBool() }]">
+      <div v-if="iconBool() && horizontalDividerBool()" class="flex-row-layout">
```

Comment:
let's add a comment here on what this container is for, since the box component has quite a few different layouts :-)

Reply:
could rename `horizontalDividerBool` as well since there's no longer one -> `seamlessHeaderBool`?

correspondingly, `verticalDividerBool` -> `seamlessNoHeaderBool`?